docs/02-app/02-api-reference/04-functions/cacheTag.mdx

```diff
@@ -1,23 +1,24 @@
 ---
 title: cacheTag
 description: Learn how to use the cacheTag function to manage cache invalidation in your Next.js application.
-version: experimental
+version: unstable
 related:
   title: Related
   description: View related API references.
   links:
     - app/api-reference/next-config-js/dynamicIO
     - app/api-reference/directives/use-cache
     - app/api-reference/functions/revalidateTag
+    - app/api-reference/functions/cacheLife
 ---
 
-The `cacheTag` function allows you to tag cached data for on-demand invalidation. By associating tags with cache entries, you can selectively purge or revalidate specific parts of your cache without affecting other cached data.
+The `cacheTag` function allows you to tag cached data for on-demand invalidation. By associating tags with cache entries, you can selectively purge or revalidate specific cache entries without affecting other cached data.
 
 ## Usage
 
-To use `cacheTag`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` and import `cacheTag` from `next/cache`:
+To use `cacheTag`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` file and import `cacheTag` from `next/cache`.
```

Comment:
Same as for the cacheLife page, we can omit the import copy as you state this below with an example. This codeblocak covers setting the flag
```suggestion
To use `cacheTag`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` file:
```